### PR TITLE
REALMC-8262: Use json document display instead of table for created resources

### PR DESCRIPTION
**Pull request by makesitgo**
~Leaving this as a draft/wip for now as we wait for #166 to be merged first~

### CODE REVIEW

File: internal/commands/app/create_test.go [hunk 1]
```diff
@@ -490,26 +484,21 @@ func TestAppCreateHandler(t *testing.T) {
...
+}
+Check out your app: cd ./remote-app && realm-cli app describe
+`, appLocal.RootDir, tc.expectedGroupID), out.String())
 
 				out.Reset()
 			})
 		}
 	})
 
+	// FIXME: these tests are incorrect, but waiting on REALMC-8830 to merge before truly fixing these
```
Comment (jandersongo) [inline internal/commands/app/create_test.go:501]: Can we remove this comment now since REALMC-8839 has been merged?

Comment (makesitgo) [reply internal/commands/app/create_test.go:501]: yes we can... thanks for catching!

**Review by nathantfrank — APPROVED** [review]
Digging the new output!

**Review by jandersongo — APPROVED** [review]
This is great! It will make writing tests so much easier 💯

**Pull request merged**